### ISSUE TITLE
Feature Request: "goto page" for shared and personal stash.

### ISSUE THREAD

**Issue by ShadwDrgn** [ChaosMarc/PlugY#55]
Would be AWESOME If i could type "/gt 100" OR "/gt NAMEOFTAB" to quickly get to a tab page!!!!!! 

This way I could type "/goto runes" or "/gt  runes" to quickly get to my runes section! or at least memorize common pages like /gt 100 if 100 is runes

Further feature functions could be a limited "Search" so that for example if i type "/gt une" it''d take me to "Runes" page if that's the first page with matching text "une" in it! this way i don't even have to fully remember the whole name! Ideally it would be case insensitive so that i don't have to memorize capitalization, but that's trivial to workaround.

**Comment by ChaosMarc** [comment]
I totally agree. Some way of jumping to specific pages is a must have in large stashes.
